libr/core/cbin.c

```diff
@@ -3459,6 +3459,24 @@ static int bin_header(RCore *r, int mode) {
 	return false;
 }
 
+static int bin_hashes(RCore *r, int mode) {
+	ut64 lim = r_config_get_i (r->config, "cfg.hashlimit");
+	RIODesc *iod = r_io_desc_get (r->io, r->file->fd);
+	if (iod) {
+		r_bin_file_hash (r->bin, lim, iod->name);
+		if (IS_MODE_JSON (mode)) {
+			r_cons_printf ("{\"");
+		}
+		r_cons_printf (r_strbuf_get (r->bin->cur->o->info->hashes));
```

Comment:
use "%s" or r_cons_strcat, but dont introduce a format string vulnerability in 2019

Reply:
Ah right I took as a puts ...